client/components/LearnerViewDirective.js

```diff
@@ -524,6 +531,9 @@ tie.directive('learnerView', [function() {
         $scope.editorContents = {
           code: ''
         };
+
+        $scope.isBrowserOnly = SERVER_URL !== null;
```

Comment:
Sorry misunderstood what "isBrowserOnly" should mean but after going over with @seanlip it works now.